tests/test_config.py

```diff
@@ -241,8 +241,19 @@ def test_config_modified_settings(self, tmp_path_home):
     ],
 )
 def test_logging(log_level, debug_level, root, instructlab, openai_httpx):
-    configure_logging(log_level=log_level, debug_level=debug_level)
+    configure_logging(log_level=log_level, debug_level=debug_level, fmt="%(message)s")
     assert logging.getLogger("root").getEffectiveLevel() == root
     assert logging.getLogger("instructlab").getEffectiveLevel() == instructlab
     assert logging.getLogger("openai").getEffectiveLevel() == openai_httpx
     assert logging.getLogger("httpx").getEffectiveLevel() == openai_httpx
+    assert logging.getLogger("root").handlers[0].formatter._fmt == "%(message)s"
+
+
+def test_invalid_log_formatter(capfd):
+    with pytest.raises(SystemExit):
+        configure_logging(log_level="INFO", debug_level=0, fmt="invalid_log_format")
+    out, err = capfd.readouterr()
+    assert (
+        "Failed to configure log format" in out
+        or "Failed to configure log format" in err
```

Comment:
why `or` here? don't we know that we log to stdout and not stderr?

Reply:
Addressed.